plugins/aea-ledger-ethereum-flashbots/setup.py

```diff
@@ -41,8 +41,8 @@
     },
     python_requires=">=3.9,<4.0",
     install_requires=[
-        "open-aea-ledger-ethereum~=1.51.0",
-        "open-aea-flashbots==1.4.0",
+        "open-aea-ledger-ethereum>=1.51.0,<2",
```

Comment:
The flashbots plugin needs to be in sync with the base ethereum plugin, so the version also needs to be same as the base plugin

Reply:
Didn't know that, sorry. Fixed back.